src/pages/Hero.tsx

```diff
@@ -43,9 +37,6 @@ export const Hero = () => {
                 getNodeStats(BTC),
             ]);
             log.debug("node", nodesRes);
```

Comment:
If you remove this log statement, we don't need to fetch `nodeRes` at all

Reply:
Done